android/libraries/rib-android/src/main/java/com/uber/rib/core/RibActivity.kt

```diff
@@ -57,4 +60,29 @@ abstract class RibActivity : AppCompatActivity() {
             super.onBackPressed()
         }
     }
+
+    override fun create(): Intent =
+        Intent()
+
+    override fun create(cls: Class<*>): Intent =
+        Intent(this, cls)
+
+
+    override fun startActivity(f: IntentCreator.() -> Intent) {
```

Comment:
Naming for "f".
Mentioned before.

Reply:
Please see my question in the other thread related to this.

Reply:
I agree with renaming
At least to `factory`

Reply:
Fixed.